Fix super-lint runs: Customise textlint, fix files and disable check of assets/scss (#361)
EGI-Federation/documentation: content/en/users/cloud-compute/ec3/apps/ecas/_index.md

![Cluster summary](ecas-summary.png)

> When the frontend Node.js of the cluster has been successfully deployed, you

Reviewer: ```suggestion
When the frontend node of the cluster has been successfully deployed, you
```

[root@oph-server ~]#
```

> Both the frontend and the working Node.js are configured by Ansible. This

Reviewer: ```suggestion
Both the frontend and the working nodes are configured by Ansible. This
```

I don't like "working" nodes, feels wrong to me, but not the point of this PR

usage and get aggregated information over time.

To access the Grafana monitoring interface, open the browser at
`https://<YOUR_CLUSTER_IP>:3000` and log in to the system using the _admin_
username and the password specified in the `.grafana_pwd` file located under the
`/root` folder.

![Grafana UI](grafana-login.png)

The Grafana-based monitoring system provides two dashboards in order to monitor
the ECAS cluster both at system and application level.

- The **infrastructure dashboard** provides information about the percentage of
  CPU, RAM, SWAP and disk used on each Node.js (the frontend and the working
  nodes).

> |                 frontend Node.js                  |            working Node.js             |

Reviewer: ```suggestion
|                 frontend node                     |            working node                |
```
